tests/integration/Dockerfile

```
@@ -0,0 +1,61 @@
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0
+
+FROM python:3.12-slim-bookworm
+
+LABEL org.opencontainers.image.authors="contextual-services-team@mozilla.com"
+
+ENV LANG=C.UTF-8
+ENV PYTHONUNBUFFERED=1
+
+ENV PATH=$PATH:/root/.cargo/bin
+ENV PYTHON_VENV=/venv
+ENV PYTEST_ARGS=""
+ENV RUST_LOG=autopush=debug,autopush_common=debug,autoendpoint=debug,autoconnect=debug,slog_mozlog_json=info,warn
```

nit. My IDE is indicating to put this value in quotes. Maybe have a look?
```suggestion
ENV RUST_LOG="autopush=debug,autopush_common=debug,autoendpoint=debug,autoconnect=debug,slog_mozlog_json=info,warn"
```